django_tables2/views.py

```diff
@@ -227,7 +227,7 @@ def get_tables_data(self):
         """
         return self.tables_data
 
-    def get_context_data(self, **kwargs: Any) -> dict[str, Any]:
+    def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
```

Comment:
Are you using python 3.8? The `dict[str, Any]` ([PEP 585](https://peps.python.org/pep-0585/)) syntax is supported by python 3.9, which is confirmed by a recent CI run: https://github.com/jieter/django-tables2/actions/runs/13018224209/job/36312532791#step:8:19